src/css.js

```diff
@@ -261,7 +260,7 @@ jQuery.extend({
 
 		// Make sure that we're working with the right name
 		name = jQuery.cssProps[ origName ] ||
-			( jQuery.cssProps[ origName ] = vendorPropName( elem.style, origName ) );
+			( jQuery.cssProps[ origName ] = vendorPropName( origName ) || origName );
```

Comment:
No test will fail without this change but we'll be needlessly checking `getComputedStyle( node ).undefined` so I added it. I assume no test is necessary for this change.